Packs/CortexXDR/Playbooks/Cortex_XDR_Alerts_Handling_v2.yml

```diff
@@ -1709,7 +1807,5 @@ inputs:
   playbookInputQuery:
 outputs: []
 tests:
-- Test XDR Playbook
+- No tests (auto formatted)
```

Comment:
you have removed the `Test XDR Playbook` - is that intended?

Reply:
yes caused you asked me in previous review ill add it again

Reply:
changed

Reply:
nope. re-added it